src/context.rs

```diff
@@ -53,6 +53,16 @@ impl<'a> ToGlibPtr<'a, *mut ffi::cairo_t> for &'a Context {
     }
 }
 
+#[cfg(feature = "use_glib")]
+impl<'a> ToGlibPtrMut<'a, *mut ffi::cairo_t> for Context {
+    type Storage = &'a mut Context;
+
+    #[inline]
+    fn to_glib_none_mut(&'a mut self) -> StashMut<'a, *mut ffi::cairo_t, Self> {
```

Comment:
If this needed for manual code, better use http://gtk-rs.org/docs/glib/translate/fn.mut_override.html

Reply:
It's for bindings generated by gir. https://github.com/etrombly/osmgpsmap-rs

Reply:
Thanks, that works well. Also since you know this well, map_point.rs line 46, is generating a f32 for an argument and it expects a *mut f32. How do I fix it with gir?

You can close this pull request if to_glib_none_mut shouldn't be used.

Reply:
It must be looks like `get_degrees`.
Seems `OsmGpsMap-1.0.gir` has error and `osm_gps_map_point_get_radians` need fix too looks like `osm_gps_map_point_get_degrees`.
Or this function can be implemented in manual way.


And please close issue yourself when no problem remains.

Reply:
Also `&mut self` for all getters better fixed with `const=true` too